model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -291,163 +306,35 @@ def _parse_output(self, metrics):
-
-        server_perf_metrics = {
-            PerfServerQueue, PerfServerComputeInput, PerfServerComputeInfer,
-            PerfServerComputeOutput
-        }
-
-        # Parse client values
-        for metric in metrics:
-            if metric not in self.perf_metrics:
-                raise TritonModelAnalyzerException(
-                    f"Perf metric : {metric} not found or supported.")
-            parse_func = getattr(self, self.perf_metrics[metric])
-            if metric in server_perf_metrics:
-                output = parse_func(server_section)
-            else:
-                output = parse_func(client_section)
-            if output is not None:
-                self._perf_records.append(output)
-
-    def _parse_perf_client_send_recv(self, section):
-        """
-        Parses Client send/recv values from the perf output
-        """
-
-        client_send_recv = None
-        if self._config['protocol'] == 'http':
-            # Http terminology
-            client_send_recv = re.search('send/recv (\d+)', section)
-        elif self._config['protocol'] == 'grpc':
-            # Try gRPC related terminology
-            client_send_recv = re.search('request/response (\d+)', section)
-        if client_send_recv:
-            client_send_recv = float(client_send_recv.group(1)) / 1e3
-            return PerfClientSendRecv(value=client_send_recv)
-        logging.warning(
-            'perf_analyzer output did not contain client send/recv time.')
-
-    def _parse_perf_client_response_wait(self, section):
-        """
-        Parses Client response wait time (network + server send/recv)
-        values from the perf output
-        """
-        client_response_wait = re.search('response wait (\d+)', section)
-        if client_response_wait:
-            client_response_wait = float(client_response_wait.group(1)) / 1e3
-            return PerfClientResponseWait(value=client_response_wait)
-        logging.warning(
-            'perf_analyzer output did not contain client response wait time.')
-
-    def _parse_perf_throughput(self, section):
-        """
-        Parses throughput from the perf analyzer output
-        """
-
-        throughput = re.search('Throughput: (\d+\.\d+e\+\d+|\d+\.\d+|\d+)',
-                               section)
-        if throughput:
-            throughput = float(throughput.group(1))
-            return PerfThroughput(value=throughput)
-        logging.warning('perf_analyzer output did not contain throughput.')
-
-    def _parse_perf_latency_avg(self, section):
-        """
-        Parses avg latency from the perf output
-        """
-        return self._parse_perf_latency(section, "Avg latency", PerfLatencyAvg)
-
-    def _parse_perf_latency_p90(self, section):
-        """
-        Parses p90 latency from the perf output
-        """
-        return self._parse_perf_latency(section, "p90 latency", PerfLatencyP90)
+        with open(self._config['model-name'], mode='r') as f:
```

Comment:
multiple places it seems that we have hardcoded the file name. Lets make it a variable or function

Reply:
Replaced with 'latency-report-file'